Dockerfile

```
@@ -2,16 +2,15 @@ FROM python:3.6-alpine3.7
 
 COPY http-server/server.py /
 
-RUN pip install cachetools
-RUN pip install logentries
-RUN pip install requests
-RUN pip install datadog
-RUN pip install semver
+RUN pip install cachetools && \
+    pip install logentries && \
+    pip install requests && \
+    pip install datadog && \
+    pip install semver
 
-RUN mkdir -p "lib/redoc-2.0.0-alpha.38"
-
-RUN wget "https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.38/bundles/redoc.standalone.js" -P "lib/redoc-2.0.0-alpha.38/"
-RUN wget "https://cdn.jsdelivr.net/npm/redoc@2.0.0-alpha.38/bundles/redoc.standalone.js.map" -P "lib/redoc-2.0.0-alpha.38/"
+RUN mkdir -p "lib/redoc-2.0.0-alpha.38" && \
```

Redoc zit inmiddels op alpha 41, ik zou 'm nog ff updaten
```suggestion
RUN mkdir -p "lib/redoc-2.0.0-alpha.41" && \
```